src/createServer.js

```diff
@@ -1,3 +1,47 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { validate } = require('./validator');
+
+const getParams = (request) => {
+  const { pathname, searchParams } = new URL(request.url, `http://${request.headers.host}`);
+  const text = pathname.slice(1);
+  const toCase = searchParams.get('toCase');
+
+  return [text, toCase];
+};
+
+const sendResponse = (response, statusCode, data) => {
+  response.setHeader('Content-Type', 'application/json');
+  response.statusCode = statusCode;
+
```

Comment:
You forgot about the status text in the responses

`If something is not correct, you should respond with 400 status, Bad request statusText`
`You should respond with status 200 and OK status text.`